src/Form/FormControl.js

```diff
@@ -11,6 +11,7 @@ export const styleSheet = createStyleSheet('FormControl', () => {
       display: 'flex',
       flexDirection: 'column',
       position: 'relative',
+      transform: 'translateZ(0)', // #6076 prevents animation glitches of children to bubble up
```

Comment:
Oh, i havent thought own layer is such an issue, not worthy of few chrome pixels.

I think its not about antialiasing, since forcing it via css does not work. Instead, its probably about animating `scale(x)` treatis content as a bitmap for performance.